package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-purchases",
-  "version": "3.0.0-beta",
+  "version": "3.0.1-beta",
```

Comment:
I'm curious about the version number, why 3.0.1 if we haven't shipped 3.0.0 yet?

Reply:
It makes it easier to know if the plugin was updated in the sample app - as in- all the newer code got pulled in and not just from a cache. I plan on moving this back to 3.0.0, it was just the easiest way for me to be sure my changes were coming through. Do you anticipate a problem with rev'ing this and then going back?